tests/unit/update-manager-test.js

```diff
@@ -223,6 +223,17 @@ describe('UpdateManager', () => {
       expect(updater.editable(doc1)).to.equal(undefined)
     })
 
+    it('Should not detect a document is editable from metadata after removeMetadata', () => {
```

Comment:
How are these tests related to the changes? I would have expected a test that assures that Collections are now working.

Reply:
They are not. It only test the new function `removeMetadata`

Reply:
the remove date includes the tricky Collection.
see issue https://github.com/linkeddata/rdflib.js/issues/631